scripts/codebuild

```diff
@@ -23,7 +23,8 @@ current_task_definition=$(aws ecs describe-services \
 aws ecs update-service \
     --cluster loadtesting \
     --service exp-web \
-    --task-definition "${current_task_definition}" \
```

Comment:
Do you mind explaining what issue this was causing again / point me to the conversation where this was originally discussed?

Reply:
The issue is in the above call to describe-services.  If there is a new task definition, the above describe-services call will retrieve the old inactive task definition revision because that is the one currently used by the service.  So I had to manually bump up the task definition if there was a newer one not yet used by the service.

I can set it back to use $current_task_definition for now if no more terraform changes are required.

https://ustcdp3.slack.com/archives/C02HCHYUU9M/p1636405974017300?thread_ts=1636405836.016800&cid=C02HCHYUU9M

Reply:
We could change the code build script to update the task definition instead of doing that through TF. That's basically what we do on MilMove except we use the AWS go API instead of the CLI.

But, let's not block on that improvement. We can always revisit in a future PR

Reply:
Yea that was more the workflow that I had seen elsewhere but it may need less frequent updates over time.